### ISSUE TITLE
add compliance audit file

### ISSUE THREAD

**Issue by MonkmanMH** [bcgov/bcgovr#65]
The bcgov github policies now require a compliance audit file; it would be handy to have a template accessible via {bcgovr}. Thanks!

**Comment by stephhazlitt** [comment]
@MonkmanMH do you mean the `yaml` file? Can you provide a link to the policy or citation?

**Comment by MonkmanMH** [comment]
Yes, the `yaml` is the one. Here's the link to the repo-mountie issue from the repo I created yesterday, that got me thinking about this feature.
https://github.com/bcgov/census-wrangling/pull/4

Additional subfeatures to consider:

* a choice option for the 4 status types
* some data science-specific guidelines, since much of our work is not "application development"

Thanks!

**Comment by stephhazlitt** [comment]
Thanks @MonkmanMH. There is a solid amount of maintenance to do on {bcgovr} in the near future, so will consider this at that time. 

In the meantime, it is 'considerate' of {repomountie} that the `COMPLIANCE.yaml` is generated for users via a PR, so at least no manual steps required to add it for now (other than specifying the category in the comments). 

I also wonder if bundling it into {bcgovr} will mean fewer users become aware of this policy step, the PR is almost an educational tool for users?

**Comment by henry-ngo** [comment]
I agree with @stephhazlitt since the repomountie PR takes care of almost everything and I found it helpful to learn that way.

But perhaps if generating that `yaml` file is an *optional* step in `use_bcgov_github()` then it could allow "advanced" users to start with the `yaml` file right away. I don't have the API off the top of my head, but one suggested implementation:  `use_bcgov_github()` can take an optional parameter such as `pia_status` and `stra_status` with the options allowed being one of the values provided by the repomountie, so that the compliance yaml file is generated based on that. If these parameters are not provided, then no yaml file is created and the repomountie will give its friendly & considerate reminder PR :)

**Comment by stephhazlitt** [comment]
I like that design, thanks @henry-ngo! 🙌
